src/test/java/core/basesyntax/strategy/handler/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.strategy.handler;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler operationHandler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandler = new BalanceOperationHandler();
+    }
+
+    @Test
+    public void operationHandler_balance_ok() {
+        Transaction transaction = new Transaction(Operation.BALANCE, new Fruit("banana"), 20);
+        operationHandler.apply(transaction);
+        assertEquals((Integer) 20, Storage.storage.get(new Fruit("banana")));
```

Comment:
```suggestion
        assertEquals(20, Storage.storage.get(new Fruit("banana")));
```
Autoboxing works without casting.
Also you can create one fruit and pass it to the both methods

Reply:
No,  it doesn't work without casting, if i will leave only 20 without castins i must to wrap second argument to 'Optional'

Reply:
you can use Integer.valueOf(20)